examples/CMakeLists.txt

```diff
@@ -20,14 +20,18 @@ add_executable(provide_prefix ${PROVIDE_PREFIX_SOURCES} ${PUBLIC_HEADERS})
 add_executable(publisher ${PUBLISHER_SOURCES} ${PUBLIC_HEADERS})
 add_executable(subscriber ${SUBSCRIBER_SOURCES} ${PUBLIC_HEADERS})
 add_executable(wampcra ${WAMPCRA_SOURCES} ${PUBLIC_HEADERS})
-add_executable(uds ${UDS_SOURCES} ${PUBLIC_HEADERS})
 add_executable(websocket_callee ${WEBSOCKET_CALLEE_SOURCES} ${PUBLIC_HEADERS})
 
+if(NOT MSVC)
+  add_executable(uds ${UDS_SOURCES} ${PUBLIC_HEADERS})
+  target_link_libraries(uds examples_parameters)
+  set(crypto_librariees crypto ssl)
+endif()
```

Comment:
Yes - on Windows they are not needed (redundant). Line 2 already adds the `${OPENSSL_LIBRARIES}` (which provides the required ssl libraries.

As for the `crypto_librariees` spelling mistake - yes, you are right. But for some reason the docker version didn't complain (it built it successfully) - though did not run them. Let me check it again.